gtsfm/runner/run_scene_optimizer.py

```diff
@@ -55,6 +55,24 @@ def run_scene_optimizer(args) -> None:
         default=20,
         help="maximum number of consecutive frames to consider for matching/co-visibility",
     )
+    parser.add_argument(
+        "--n_workers",
```

Comment:
I think num_workers sounds better, but up to you.

Reply:
Thanks Akshay. i like that -- just updated it.